WORKSPACE

```diff
@@ -38,7 +38,7 @@ bind(
 # Commit date: 2023-01-27
 ENVOY_SHA = "ef10c60ef8ff79061116471d7786faefe94f6fcb"
 
-ENVOY_SHA256 = "6f068cd710c391049134a13b1dac9e104a624b411f0316479b092d2038fa3ad4"
+ENVOY_SHA256 = "6a58379aea24b053c411f4d7bda486e9e6665dc75042459f22d1a8ba443d11c9"
```

Comment:
Seems strange the SHA256 was updated without the SHA.

Reply:
Github-wide problem.